packages/react-charts/src/components/ChartBoxPlot/examples/ChartBoxPlot.md

```diff
@@ -0,0 +1,354 @@
+      }
+      return `${datum.y !== null ? datum.y : 'no data'}`;
+    }
+    return (
+      <div style={{ height: '350px', width: '600px' }}>
+        <Chart
+          ariaDesc="Average number of pets - possibly more information to summarize the data in the chart."
+          ariaTitle="Embedded legend example chart title"
+          containerComponent={
+            <CursorVoronoiContainer
+              cursorDimension="x"
+              labels={({ datum }) => labelFormatter(datum)}
+              labelComponent={
+                <ChartLegendTooltip
+                  boxPlotData={{
+                    max: { label: 'max' },
+                    median: { label: 'mdn' },
+                    min: { label: 'min' },
+                    q1: { label: 'q1', isVisible: false },
+                    q3: { label: 'q3', isVisible: false }
+                  }}
+                  legendData={legendData}
+                  title={(datum) => datum.x}
+                />
+              }
+              mouseFollowTooltips
+              voronoiDimension="x"
+              voronoiPadding={50}
+            />
+          }
+          domain={{y: [0, 13]}}
+          domainPadding={{ x: [30, 25] }}
+          legendData={legendData}
+          legendPosition="bottom"
+          height={350}
+          name="chart5"
+          padding={{
+            bottom: 75, // Adjusted to accommodate legend
+            left: 50,
+            right: 50,
+            top: 50
+          }}
+          themeColor={ChartThemeColor.green}
+          width={600}
+        >
+          <ChartAxis />
+          <ChartAxis dependentAxis showGrid />
+          <ChartBoxPlot
+            data={[
+              { name: 'Cats', x: '2015', y: [1, 2, 3, 5] },
+              { name: 'Cats', x: '2016', y: [3, 2, 8, 10] },
+              { name: 'Cats', x: '2017', y: [2, 8, 6, 5] },
+              { name: 'Cats', x: '2018', y: [1, 3, 2, 9] }
+            ]}
+            name="cats"
+          />
+          <ChartThreshold
+            data={[
+              { name: 'Limit', x: '2015', y: 12 },
+              { name: 'Limit', x: '2016', y: 12 },
+              { name: 'Limit', x: '2017', y: 12 },
+              { name: 'Limit', x: '2018', y: 12 }
+            ]}
+            name="limit"
+            style={{
+              data: {
+                stroke: chart_color_orange_300.var
+              }
+            }}
+          />
+        </Chart>
+      </div>
+    );
+  }
+}
+```
+
+### Embedded HTML
+
+This demonstrates how to embed HTML within a tooltip. Combining cursor and voronoi containers is required to display tooltips with a vertical cursor.
+
+```js
+import React from 'react';
+import { Chart, ChartAxis, ChartBoxPlot, ChartCursorFlyout, ChartCursorTooltip, ChartThemeColor, createContainer } from '@patternfly/react-charts';
+// import '@patternfly/patternfly/patternfly-charts.css'; // Required for mix-blend-mode CSS property
```

Comment:
Is this intended  as a note to help consumers or did you mean to remove it?

Reply:
This is included in all our chart examples. It is there in case devs want chart CSS use mix-blended-mode